CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 
 - Removed support for `cxx_type` and `cxx_return_type` and related conversion methods.
+- Removed `newCppObject` function that allowed creation of default-constructed QObject from Rust.
```

Comment:
add an entry in here for adding the contructor trait which allows support for
- custom arguments
- base arguments (which unlocks implementing types like QQuickPaintedItem)
- initialise method for starting things in Rust at the constructor (like threads)

Reply:
Let's post-pone this until the Constructor trait is more thoroughly tested.
Then we'll add documentation for the feature as well.